UnitTests/CatalogTests/ServiceTests/CommentsServiceTest/AddCommentTest.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Catalog.Services.Comments;
+using Common.Interfaces;
+using Common.Models.Comments;
+using FluentValidation;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using System.Threading.Tasks;
+
+namespace UnitTests.CatalogTests.ServiceTests.CommentsServiceTest
+{
+
+    [TestClass]
+    public class AddCommentTests
+    {
+        private Mock<IDataAccessor<Comment>> CommentsAccessorMock;
+        private CommentsRepository Service;
+
+        public AddCommentTests()
+        {
+            CommentsAccessorMock = new Mock<IDataAccessor<Comment>>();
+            Service = new CommentsRepository(CommentsAccessorMock.Object);
+        }
+
+        [TestMethod]
+        public void ValidComment_ReturnsTrue()
+        {
+            // Arrange
+            var id = 1;
+            var expectedResult = true;
+
+            CommentsAccessorMock
```

Comment:
Something went wrong with this test. Class states that you are testing 'AddComment' method, while you are mocking Deletion db method and then testing DeleteComment method.
This test should be reviewed and rearranged.

- [ ] Rename and sort test methods to be in the proper test classes.